File: Somi Jeon/2022-06-06-Two-Sum.js
const twoSum = function (nums, target) {
  let hashtable = new Map();
  for (let i = 0; i < nums.length; i++) {
    let diff = target - nums[i];

    if (hashtable.hasOwnProperty(diff)) {
      return [hashtable[diff], i];
    }

    hashtable[nums[i]] = i;
  }

  return null;
};

File: Jonghyeok Kim/2022-06-06-two-sum.py
from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hash_map = {}
        for index in range(len(nums)):
            diff = target - nums[index]
            if diff in hash_map:
                return [index, hash_map[diff]]
            else:
                hash_map[nums[index]] = index
                